### DISCUSSION TITLE
网站配置中的性能调整中的超时时间不生效

### DISCUSSION THREAD

**Discussion by cktk** [1Panel-dev/1Panel#6421] [Q&A]
### 1Panel 版本

v1.10.14-lts

### 问题描述

网站配置中的性能调整中的超时时间不生效  

<img width="1329" alt="image" src="https://github.com/user-attachments/assets/612d71b1-6c92-44d3-a18e-3009eba056a2">
<img width="313" alt="image" src="https://github.com/user-attachments/assets/bf14a47d-c3f7-43da-b6bf-8d06b05c9799">
<img width="299" alt="image" src="https://github.com/user-attachments/assets/85333527-98e9-43bc-bb5b-2432c6f01766">

### 重现步骤

调整超时时间    重启 
调用超时的接口  

### 相关日志输出

```shell
<img width="830" alt="image" src="https://github.com/user-attachments/assets/3483321c-2e91-4b09-a8e2-183102cec8b9">
<img width="1846" alt="image" src="https://github.com/user-attachments/assets/893e56c2-cfaa-4d34-90cb-47d21f224a63">
```

**Comment by cktk** [comment]
<img width="830" alt="image" src="https://github.com/user-attachments/assets/3483321c-2e91-4b09-a8e2-183102cec8b9">
<img width="1846" alt="image" src="https://github.com/user-attachments/assets/893e56c2-cfaa-4d34-90cb-47d21f224a63">

**Comment by cktk** [comment]
经过测试 使用IP直接访问的情况下无超时的情况

<img width="414" alt="image" src="https://github.com/user-attachments/assets/07464bbb-db3e-4b42-af58-480ee6324bcf">

**Comment by wanghe-fit2cloud** [comment]
感谢反馈，我们先在本地环境复现一下上述问题。

**Comment by zhengkunwang223** [comment]
keepalive_timeout 主要作用于请求处理完成之后的空闲连接，而不是控制请求本身的超时时间
具体请看 https://nginx.org/en/docs/http/ngx_http_core_module.html#keepalive_timeout

**Reply by cktk** [reply]
有没有调整请求本身超时时间的配置
